implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -216,12 +223,14 @@ impl CreateCommand {
             || self.config_args.enrollment_ticket.is_some()
     }
 
-    /// Return true if the `name` argument is not a config file path or URL
+    /// Return true if the `name` argument is not a URL, a file path, or an inline config
```

Comment:
The comment could say that it is just a name if it's not a URL, a file path, or an inline config.

Reply:
I think I will rename the function from `has_name_arg` to `name_arg_is_a_config`. Sounds more explicit to me. What do you think? The comment would also look less confusing.